install/kubernetes/helm/istio/charts/gateways/templates/deployment.yaml

```diff
@@ -1,18 +1,16 @@
 {{- range $key, $spec := .Values }}
 {{- if ne $key "enabled" }}
 {{- if $spec.enabled }}
+
+{{- $labels := dict "release" $.Release.Name "chart" (include "gateway.chart" $) "heritage" $.Release.Service }}
+{{- $labels = merge $labels $spec.labels }}
```

Comment:
There is an helm issue that complains the `=` here, which has been fixed from helm `v2.13.0`, see: https://github.com/helm/helm/issues/4987

Reply:
Ahh thanks! thats why it worked locally then.

We advertise helm 2.10 support so probably need to find an even dirtier hack to accomplish this